esmvaltool/recipes/examples/recipe_check_obs.yml

```diff
@@ -202,6 +202,23 @@ diagnostics:
     scripts: null
 
 
+  GPCC:
+    description: GPCC check
+    variables:
+      tas:
```

Comment:
Is this correct? In the configuration file only precipitation is mentioned.

Reply:
No, there is only precipitation.

Reply:
```suggestion
```